diff -ruN a/liquidctl/driver/nzxt_smart_device.py b/liquidctl/driver/nzxt_smart_device.py
--- a/liquidctl/driver/nzxt_smart_device.py	2019-11-10 12:07:55.708305528 +0300
+++ b/liquidctl/driver/nzxt_smart_device.py	2019-11-10 11:57:02.746514369 +0300
@@ -316,6 +316,10 @@
             'speed_channel_count': 0,
             'color_channel_count': 4
         }),
+	(0x1e71, 0x2002, None, 'NZXT HUE 2 V2 (experimental)', {
+            'speed_channel_count': 0,
+            'color_channel_count': 4
+        }),
     ]
 
     _READ_LENGTH = 64
